### ISSUE TITLE
Implement ansible connection backend and Ansible module

### ISSUE THREAD

**Issue by philpep** [philpep/testinfra#35]
Ex:

```
testinfra --ansible-inventory=/my/inventory --connection=ansible --hosts=host1,host2 tests
```

```
>>> Ansible("file", "path=/etc/passwd")
{u'group': u'root', u'uid': 0, u'changed': False, 'invocation': {'module_name': 'file', 'module_complex_args': {}, 'module_args': u'path=/etc/passwd'}, u'state': u'file', u'gid': 0, u'mode': u'0644', u'owner': u'root', u'path': u'/etc/passwd', u'size': 2590}
```

This can be done with `ansible.runner.Runner` python API

**Comment by ktosiek** [comment]
There's already pytest-ansible: https://github.com/jlaska/pytest-ansible which provides that, maybe that's good enough?

**Comment by philpep** [comment]
I've a WIP in #36 . Indeed pytest-ansible looks interesting, maybe there is good idea to take from it.

**Comment by philpep** [comment]
Merged in #36
